Incorporate data from & critiques of SOLIDARITY trial (#743)
greenelab/covid19-review: content/20.pharmaceuticals.md

Another study comparing 5- and 10-day treatment regimens had similar results but was also limited because of the lack of a placebo control [@doi:10.1056/NEJMoa2015301].
The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results provided motivation for placebo-controlled studies.

Remdesivir was later tested in a double-blind placebo-controlled phase 3 clinical trial performed at 60 trial sites, 45 of which were in the United states [@doi:10.1056/NEJMoa2007764; @clinicaltrials:NCT04280705].
The trial recruited 1,062 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
Patients were stratified for randomization based on site and the severity of disease presentation at baseline [@doi:10.1056/NEJMoa2007764].
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
Data was analyzed from a total of 1,059 patients who completed the 29 day course of the trial (517 assigned to remdesivir and 508 to placebo) [@doi:10.1056/NEJMoa2007764].
The two groups were well matched demographically and clinically at baseline.

> Those who received remdesivir had a median recovery time of 10 days, as compared with 15 days in those who received placebo (rate ratio for recovery, 1.29; 95% CI, 1.12 to 1.49; P<0.001).

Reviewer: ```suggestion
Those who received remdesivir had a median recovery time of 10 days, as compared with 15 days in those who received placebo (rate ratio for recovery, 1.29; 95% CI, 1.12 to 1.49; _p_ < 0.001).
```

As of October 22, 2020, remdesivir received FDA approval based on three clinical trials [@url:https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19].

However, results suggesting no effect of remdesivir on survival were reported by the WHO Solidarity trial [@doi:10.1056/NEJMoa2023184].
This large-scale, open-label trial enrolled 11,330 adult in-patients at 405 hospitals in 30 countries around the world [@doi:10.1056/NEJMoa2023184].
Patients were randomized in equal proportions into four experimental and a control conditions, corresponding to four candidate treatments for COVID-19 and SOC, respectively; no placebo was administered.
The 2750 patients in the remdesivir group were administered 200 mg intravenously on the first day and 100 mg on each subsequent day until day 10 and assess for in-hospital death (primary endpoint), duration of hospitalization, and progression to mechanical ventilation.
There were also 2708 control patients who would have been eligible and able to receive remdesivir were they not assigned to the control group.
A total of 604 patients among these two cohorts deceased during initial hospitalization.

> The rate ratio of death between these two groups was therefore not significant (p = 0.50), suggesting that the administration of remdesivir did not affect survival.

Reviewer: ```suggestion
The rate ratio of death between these two groups was therefore not significant (_p_ = 0.50), suggesting that the administration of remdesivir did not affect survival.
```

Reviewer: I’ll let you make a decision on what way to write the p values for consistency - @rando2

The two secondary analyses similarly did not find any effect of remdesivir.
Additionally, the authors compared data from their study with data from three other studies of remdesivir (including [@doi:10.1056/NEJMoa2007764]) stratified by supplemental oxygen status.
The adjusted rate ratio for death based on this meta-analysis was 0.91.
These results thus do not support the previous findings that remdesivir reduced median recovery time and mortality risk in COVID-19 patients.

> In response to the results Solidarity trial, Gilead, which manufactures remdesivir under the name Veklury^R, released a statement pointing to the fact that the Solidarity trial was not placebo-controlled or double-blind and at the time of the statement had not been peer reviewed [@url:https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial]; these sentiments have been echoed elsewhere [@doi:10.1093/eurheartj/ehaa934].

Reviewer: ```suggestion
In response to the results of the Solidarity trial, Gilead, which manufactures remdesivir under the name Veklury^R, released a statement pointing to the fact that the Solidarity trial was not placebo-controlled or double-blind and at the time of the statement had not been peer reviewed [@url:https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial]; these sentiments have been echoed elsewhere [@doi:10.1093/eurheartj/ehaa934].
```

> Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe case [@url:https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug; @doi:10.1056/NEJMe2034294].

Reviewer: ```suggestion
Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe cases [@url:https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug; @doi:10.1056/NEJMe2034294].
```

However, the publication associated with the trial sponsored by Gilead did suggest an effect of remdesivir on patients with severe disease, identifying an 11 versus 18 day recovery period (rate ratio for recovery: 1.31, 95% CI 1.12 to 1.52), although the results of a significance test were not provided [@doi:10.1056/NEJMoa2007764].

> Additionally, a smaller analysis comparing treatment with remdesivir to SOC in patients with moderate COVID-19 reported a small effect size, corresponding to minimal differences between these groups at day 11 [@doi:10.1001/jama.2020.16349].

Reviewer: ```suggestion
Additionally, a smaller analysis comparing treatment with remdesivir to standard of care in patients with moderate COVID-19 reported a small effect size, corresponding to minimal differences between these groups at day 11 [@doi:10.1001/jama.2020.16349].
```

Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.

#### Conclusions of the Present Analysis

Due to the large number of clinical trials currently under examination (Figure @fig:ebm-trials), not all candidates are examined here.
Instead, this review seeks to provide an overview of the range of mechanisms that have been explored and to examine some prominent candidates in the context of the pathogenesis of and immune response to SARS-CoV-2.
As more research becomes available, this review will be updated to include additional therapeutics that emerge and to include new findings that are released about those discussed here.
While no therapeutics or vaccines were developed for SARS-CoV-1 or MERS-CoV, the current state of COVID-19 research suggests that the body of literature produced before and after the emergence of these viruses has prepared the biomedical community for a rapid response to novel HCoV like SARS-CoV-2.
At present, two pharmaceutical therapeutics and one vaccine have been authorized for the treatment and prevention, respectively, of COVID-19.

> As the COVID-19 pandemic continues to be a topic of significant worldwide interest, more information is expected to become available about pharmaceutical mechanisms that can be used to combat this, and possibly other, HCoV.

Reviewer: ```suggestion
As the COVID-19 pandemic continues to be a topic of significant worldwide concern, more information is expected to become available about pharmaceutical mechanisms that can be used to combat this, and possibly other, HCoV.
```
